src/rust/cryptography-x509/src/extensions.rs

```diff
@@ -231,7 +231,32 @@ pub struct BasicConstraints {
 
 pub type SubjectAlternativeName<'a> = asn1::SequenceOf<'a, name::GeneralName<'a>>;
 pub type IssuerAlternativeName<'a> = asn1::SequenceOf<'a, name::GeneralName<'a>>;
-pub type ExtendedKeyUsage<'a> = asn1::SequenceOf<'a, asn1::ObjectIdentifier>;
+
+pub struct ExtendedKeyUsage<'a>(asn1::SequenceOf<'a, asn1::ObjectIdentifier>);
+
+impl<'a> Iterator for ExtendedKeyUsage<'a> {
+    type Item = asn1::ObjectIdentifier;
+
+    fn next(&mut self) -> Option<Self::Item> {
+        self.0.next()
+    }
+}
```

Comment:
NB: Added this to avoid making the newtype's inner member public just for iteration, but I could save these ~5 lines by doing that instead 🙂